x-pack/plugins/upgrade_assistant/server/routes/node_disk_space.ts

```diff
@@ -90,7 +90,7 @@ export function registerNodeDiskSpaceRoute({ router, lib: { handleEsError } }: R
               );
 
               // If the percentage available is < the low disk watermark setting, mark node as having low disk space
-              if (percentageAvailable < rawLowDiskWatermarkPercentageValue) {
+              if (percentageAvailable < 100 - rawLowDiskWatermarkPercentageValue) {
```

Comment:
[nit] I might pull this out to its own variable and add a code comment or update the existing one, so it's a little more clear.

Something like this (feel free to rename and/or adjust the comment)

```
const freeDiskSpaceAllocated = 100 - rawLowDiskWatermarkPercentageValue // ES interprets this setting as a threshold for used disk space; we want free disk space

if (percentageAvailable < freeDiskSpaceAllocated) {...}
```

Reply:
Love it, will patch it up